daemon/algod/api/server/v2/handlers.go

```diff
@@ -56,9 +58,25 @@ type Handlers struct {
 	Shutdown <-chan struct{}
 }
 
+// LedgerForAPI describes the Ledger methods used by the v2 API.
+type LedgerForAPI interface {
```

Comment:
EncodedBlockCert is a transitive dependency because v2 API calls rpcs.RawBlockBytes and passes it a ledger that needs to support EncodedBlockCert.. agreed that pkgname.LedgerForPkgName is kind of redundant, I would also be happy to name the interfaces v2.Ledger and BlockService.Ledger in keeping with https://github.com/golang/go/wiki/CodeReviewComments#interfaces